Add removal of account console v1 to release notes (#21212)
keycloak/keycloak: docs/documentation/release_notes/topics/22_0_0.adoc

In scenarios where advanced management is needed you may now directly update most fields on operator managed resources that have not been set by the operator directly. This can be used as an alternative to the unsupported stanza of the Keycloak spec. Like the unsupported stanza these direct modifications are not considered supported. If your modifications prevent the operator from being able to manage the resource, there Keycloak CR will show this error condition and the operator will log it.

> = Account Console v1

Reviewer: Perhaps mention the removal in the heading as well?
```suggestion
= Account Console v1 removal
```

> The old Account Console (v1) is now completely removed. This version of the Account Console was marked as deprecated

Reviewer 2: ```suggestion
Account Console is now completely removed. This version of the Account Console was marked as deprecated
```

Reviewer: I still think we need to refer to the v1 explicitly, we currently maintain 3 different versions, so users need to know what we're talking about.

Reviewer 2: Okay,  then "The Account Console v1"
You don't need the parentheses if you don't provide them eslewhere.

Author: We used to refer to it as the "old account console", but more recently refer to it as "account console v1". Hence, why I used "old Account Console (v1)".

Reviewer 2: Okay, I think that works too, @stianst

> back in Keycloak 12.

Reviewer 2: ```suggestion
in Keycloak 12.
```

> The initial plans was to remove it much earlier, but was kept for two reasons. Firstly, to provide

Reviewer 2: ```suggestion
The initial plan was to remove it much sooner, but it was retained for two reasons: 

* We want to provide
```
